config/application.rb

```diff
@@ -16,6 +16,12 @@
 
 module TipsyTally
   class Application < Rails::Application
+    config.before_configuration do
+      env_file = File.join(Rails.root, 'config', 'local_env.yml')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.